climada/hazard/centroids/centr.py

```diff
@@ -491,52 +491,98 @@ def clear(self):
         self.__init__()
 
     def append(self, centr):
-        """Append raster or points.
+        """Append centroids points.
+
+        If centr is raster it is converted to points first using
```

Comment:
```suggestion
        If self or centr are rasters they are converted to points first using
```

Reply:
Also probably worth mentioning this is a wrapper around `Centroids.union` and pointing to that documentation

Reply:
Done here 
173533f